modules/gitops-installing-argo-rollouts-cli-by-using-mac-os.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-cli-on-mac-os_{context}"]
+= Installing Argo Rollouts CLI on Mac OS
+
+If you are a macOS user, you can install the Argo Rollouts CLI by using the link:https://brew.sh[Homebrew] package manager.
+
+.Prerequisites
+
+* You have installed the Homebrew (`brew`) package manager.
+
+.Procedure
+
+* Run the following command to install the Argo Rollouts CLI:
++
+[source,terminal]
+----
+$ brew install argoproj/tap/kubectl-argo-rollouts
+----
```

Comment:
Is it worth including verification instructions beyond this point? The Linux module does that.

Reply:
I agree that the Linux module had some additional steps but during the walkthrough provided by the SME for this module, we did not have additional steps after the installation was completed.